_documentation/src/docs/documentation/10_user-guide.adoc

```diff
@@ -13,3 +13,5 @@ On any merge request, if a user post a review (or a comment) containing "🔧" (
 image::needs-work-comment.png[alt=Flower,width=788,height=175]
 
 Using the "New changes since the comment." link, you can access the diff view where you compare the changes since your last review.
+
+NOTE: Comments or threads on the "Changes" tab are ignored (when you post a remark on the code in the diff view). The "needs work" message has to be posted on the review dialog (or as comment in the "Overview" tab).
```

Comment:
Good point I need to verify it.

Reply:
If you start a thread in the "Overview" tab, the type is `DiscussionNote`, so with the new filter it will be ignored.

Reply:
But should it be ignored? If the reviewer only has a general remark (not specific to any line of the diff), he would start a thread from the overview, but then it would be ignored...

Reply:
And if he creates a thread/comment in the diff view and do not write "needs work" in the review dialog, then it will also be ignored.

I think this is fine, since our process is to write "needs work" in the review dialog.